src/docs/transactions.dox

```diff
@@ -148,4 +148,27 @@ WT_SESSION::snapshot with a configuration that includes
 the semantics supported by the drop configuration.
 
 Named snapshots are not durable: they do not survive WT_CONNECTION::close.
+
+@section transaction_timestamps Timestamps
+
+Applications can assign transactions explicit commit timestamps, then read
+as-of a timestamp.  Timestamps are communicated to WiredTiger as hexadecimal
+values with a length configurable at build time (default 8 bytes).
```

Comment:
The application doesn't know (or care) how we store a timestamp inside WiredTiger, all the application knows is the maximum size of the string it's going to pass in, so that's what the application can configure.

Imagine an application timestamp that's a maximum of 19 bytes. What calculation should an application writer perform to decide what to use as the "length configurable at build time"?